app/assets/javascripts/angular/modules/truncate.js

```diff
@@ -0,0 +1,16 @@
+angular.module('truncate', []).filter('characters', function () {
```

Comment:
Truncate is a new helper to return a specified number of characters from a block of text. We use it for message previews.